app/assets/stylesheets/dashboard.css

```diff
@@ -0,0 +1,651 @@
+/* Override for default agg status icon color */
+/* line 3, scss/patternfly-overrides.scss */
+.card-pf-status .card-pf-title .pficon, .card-pf-status .fa, .card-pf-aggregate-status .card-pf-title .pficon, .card-pf-aggregate-status .fa {
+    color: #1186C1 !important;
+    margin-top: 7px;
+}
+
+/* line 8, scss/patternfly-overrides.scss */
+.data-list-pf .list-group-item.active .pficon, .data-list-pf .list-group-item.active:hover .pficon, .data-list-pf .list-group-item.active:focus .pficon {
+    color: #1186C1;
+}
+
+/* line 11, scss/patternfly-overrides.scss */
+.data-list-pf .list-group-item .pficon {
+    font-size: 18px;
+}
+
+/* line 14, scss/patternfly-overrides.scss */
+.tiles-view-pf .tile.active .pficon, .tiles-view-pf .tile.active:hover .pficon, .tiles-view-pf .tile.active:focus .pficon {
+    color: #1186C1;
+}
+
+/* line 2, scss/main.scss */
+.pf-fixed-body {
+    padding-top: 125px;
+}
+
+/* line 10, scss/main.scss */
+.miq-body {
+    background: #f5f5f5;
+}
+
+/* line 14, scss/main.scss */
+.card-pf .loading {
+    margin: 10px auto;
+}
+
+/* line 18, scss/main.scss */
+.card-pf-providers {
+    min-height: 144px;
+}
+/* line 21, scss/main.scss */
+.card-pf-providers .providers {
+    padding: 20px 0;
+}
+/* line 24, scss/main.scss */
+.card-pf-providers .providers a {
+    color: #000000;
+    text-decoration: none;
+}
+/* line 29, scss/main.scss */
+.card-pf-providers .providers a:hover {
+    color: #00618a;
+    text-decoration: underline;
+}
+/* line 33, scss/main.scss */
+.card-pf-providers .providers .count {
+    margin: 0 5px;
+}
+/* line 37, scss/main.scss */
+.card-pf-providers .providers .provider-icon {
+    font-size: 22px;
+    position: relative;
+    top: 5px;
+    padding-right: 5px;
+    opacity: .9;
+}
+/* line 44, scss/main.scss */
+.card-pf-providers .providers .provider-icon:hover {
+    opacity: 1;
+}
+/* line 50, scss/main.scss */
+.card-pf-providers .provider-icon-large {
+    font-size: 100px;
+    position: relative;
+    padding-bottom: 4px;
+    width: 100%;
+    text-align: center;
+}
+
+/* line 59, scss/main.scss */
+.providers-status {
+    padding-bottom: 9px !important;
+}
+
+/* line 63, scss/main.scss */
+.pf-obj-status-large .card-pf-title {
+    padding-top: 24px;
+    padding-bottom: 24px;
+}
+
+/* line 69, scss/main.scss */
+.pficon-kubernetes::before {
```

Comment:
pf-obj-status-large is a leftover from some prototyping work and can be removed.

There are a few other overrides that are necessary but @himdel is correct, we should have a parent selector for the specific dashboard area to avoid unintentional overrides.